sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/InMemoryPartitionManager.java

```diff
@@ -20,30 +20,52 @@
     private final Map<String, PartitionOwnership> partitionOwnershipMap = new ConcurrentHashMap<>();
     private final ClientLogger logger = new ClientLogger(InMemoryPartitionManager.class);
 
+    /**
+     * {@inheritDoc}
+     *
+     * @param eventHubName The Event Hub name to get ownership information.
```

Comment:
This reads awkwardly to me.   Maybe something like "The name of the Event Hub to list ownership of"?